bzero is deprecated in POSIX.1-2001 and removed for POSIX.1-2008 so
switch to memset instead.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -purN pptp-1.10.0.orig/pptp_callmgr.c pptp-1.10.0/pptp_callmgr.c
--- pptp-1.10.0.orig/pptp_callmgr.c	2018-01-18 01:12:00.000000000 +0100
+++ pptp-1.10.0/pptp_callmgr.c	2018-01-31 12:18:09.213534169 +0100
@@ -340,7 +340,7 @@ int open_inetsock(struct in_addr inetadd
     }
 #endif
     if (localbind.s_addr != INADDR_NONE) {
-        bzero(&src, sizeof(src));
+        memset(&src, 0, sizeof(src));
         src.i.sin_family = AF_INET;
         src.i.sin_addr   = localbind;
         if (bind(s, &src.a, sizeof(src.i)) != 0) {
diff -purN pptp-1.10.0.orig/pptp_gre.c pptp-1.10.0/pptp_gre.c
--- pptp-1.10.0.orig/pptp_gre.c	2018-01-18 01:12:00.000000000 +0100
+++ pptp-1.10.0/pptp_gre.c	2018-01-31 12:18:09.213534169 +0100
@@ -109,7 +109,7 @@ int pptp_gre_bind(struct in_addr inetadd
     }
 #endif
     if (localbind.s_addr != INADDR_NONE) {
-        bzero(&loc_addr, sizeof(loc_addr));
+        memset(&loc_addr, 0, sizeof(loc_addr));
         loc_addr.i.sin_family = AF_INET;
         loc_addr.i.sin_addr   = localbind;
         if (bind(s, &loc_addr.a, sizeof(loc_addr.i)) != 0) {
